src/Bridge/Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -103,7 +103,9 @@ public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGenerator
             throw new InvalidArgumentException('Item per page parameter should not be less than 0');
         }
 
-        $page = $this->getPaginationParameter($request, $this->pageParameterName, 1);
+        if (0 >= $page = $this->getPaginationParameter($request, $this->pageParameterName, 1)) {
+            $page = 1;
```

Comment:
what about throwing an error instead?

Reply:
@soyuka, it can be useful if the front call the api with empty `page` param for first page: `/resources?page=`

Reply:
> it can be useful if the front call the api with empty `page` param for first page: `/resources?page=`

I'm actually surprised that we're not casting to integer somewhere...